llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -6971,7 +6971,7 @@ static bool hasPassthruOp(unsigned Opcode) {
          Opcode <= RISCVISD::LAST_STRICTFP_OPCODE &&
          "not a RISC-V target specific op");
   static_assert(
-      RISCVISD::LAST_VL_VECTOR_OP - RISCVISD::FIRST_VL_VECTOR_OP == 134 &&
+      RISCVISD::LAST_VL_VECTOR_OP - RISCVISD::FIRST_VL_VECTOR_OP == 139 &&
       RISCVISD::LAST_STRICTFP_OPCODE - RISCVISD::FIRST_STRICTFP_OPCODE == 21 &&
       "adding target specific op should update this function");
   if (Opcode >= RISCVISD::ADD_VL && Opcode <= RISCVISD::VFMAX_VL)
```

Comment:
Shouldn't this return true for the added nodes?

Reply:
The routines hasPassthruOp and hasMaskOp are only used by lowerToScalableOp and lowerVPOp.  These new nodes don't go through either of those, and thus the result is a don't care.  Honestly, the code structure of these routines could use some cleanup, but it's not a problem with the patch.

Reply:
> Honestly, the code structure of these routines could use some cleanup

Those will be removed by #138381. Just wanted to make sure the `HasPassthruOp` flag should be set on those nodes.
CC @lenary